### ISSUE TITLE
integrate tor into the OS

### ISSUE THREAD

**Issue by PhoenixLandPirate** [ubports/ubuntu-touch#843]
Tor allows for more private internet browsing, orbot allows you to choose apps that can connect via tor VPN, having tor as a default option in the VPN selection would make it easy for people to connect to the internet a bit more privately.

**Comment by 3arn0wl** [comment]
:) This is something I would like to see too.

**Comment by Spraltik** [comment]
+1 here also. It was for me a matter of privacy switching to Ubuntu Touch so that I would be running an operating system without any google influence. But having no Tor Browser makes me wonder if it's the right choice.

**Comment by deadblackclover** [comment]
I really want to have Tor, since now site blocking is very active. He also wants to protect his traffic and data.

**Comment by PhoenixLandPirate** [comment]
Yeah in the UK you have to send a picture of your ID or use a VPN to access archive.org for example.
